fecgov/fec-cms
# [MERGE WITH GIT FLOW] [HOTFIX] Add more candidates to manual conversion list

## Summary (required)

- Resolves #3998
Add more candidates to manual conversion list

## Impacted areas of the application

List general components of the application that this PR will affect:

-  Add more candidates to manual conversion list
- Committee and candidate profile pages should have explanatory text:

https://www.fec.gov/data/committee/C00727008
https://www.fec.gov/data/committee/C00712224
https://www.fec.gov/data/committee/C00706564
https://www.fec.gov/data/committee/C00699090

https://www.fec.gov/data/candidate/H0CA25170
https://www.fec.gov/data/candidate/H0OH04085
https://www.fec.gov/data/candidate/H0GA07265
https://www.fec.gov/data/candidate/P00010793

## Related PRs

List related PRs against other branches:

https://github.com/fecgov/fec-cms/pull/3953

## How to test

- Would like two approvals, at least one developer please

- If you can run locally: Committee and candidate profile pages should have explanatory text:

http://localhost:8000/data/committee/C00727008
http://localhost:8000/data/committee/C00712224
http://localhost:8000/data/committee/C00706564
http://localhost:8000/data/committee/C00699090

http://localhost:8000/data/candidate/H0CA25170
http://localhost:8000/data/candidate/H0OH04085
http://localhost:8000/data/candidate/H0GA07265
http://localhost:8000/data/candidate/P00010793

- Anyone: Make sure the values match the [tracking spreadsheet](https://docs.google.com/spreadsheets/d/1VypSyRziT-_bgf9PslVNCkVUZ15GovF33AshZ9-f-4s/edit#gid=0)

Note: There's an issue with Beto's 2017-2018 tab 404-ing, noted here: https://github.com/fecgov/fec-cms/issues/3805#issuecomment-680957042
